hummingbot/templates/conf_pure_market_making_strategy_TEMPLATE.yml

```diff
@@ -106,5 +106,8 @@ price_source_custom: null
 #Take order if they cross orderbook when external price source is enabled
 take_if_crossed: null
 
+#Order override to override orders placed by order_amount and order_level parameter
```

Comment:
You can add an example on how this new parameter can be configured (in a comment)